src/SignalR/samples/SignalRSamples/Hubs/Chat.cs

```diff
@@ -9,58 +9,48 @@ namespace SignalRSamples.Hubs
 {
     public class Chat : Hub
     {
-        public override async Task OnConnectedAsync()
+        public Task Send(string name, string message)
         {
-            await Clients.All.SendAsync("Send", $"{Context.ConnectionId} joined");
+            return Clients.All.SendAsync("Send", $"{name}: {message}");
         }
 
-        public override async Task OnDisconnectedAsync(Exception ex)
+        public Task SendToOthers(string name, string message)
         {
-            await Clients.Others.SendAsync("Send", $"{Context.ConnectionId} left");
```

Comment:
Can we include the user name in the query string or something so we can keep the user joined/left messages?